can_tools/scrapers/official/IA/ia_vaccine_county.py

```diff
@@ -0,0 +1,95 @@
+import pandas as pd
+import us
+import requests
+import json
+from io import StringIO
+from can_tools.scrapers.official.base import StateDashboard, CMU
+
+class IowaCountyVaccine(StateDashboard):
+    has_location = False
+    location_type = "county"
+    state_fips = int(us.states.lookup("Iowa").fips)
+    source = "https://coronavirus.iowa.gov/pages/vaccineinformation#VaccineInformation"
+    source_name = "Iowa Department of Public Health"
+
+    dashboard_link = "https://public.domo.com/embed/pages/1wB9j"
+    card_id = "1179017552"
+    csv_link = "https://public.domo.com/embed/pages/1wB9j/cards/1179017552/export"
+    # Don't know if this will need to be renewed...
+    # If it does, to get this token: 
+    # 1) open the dashboard_link in a browser
+    # 2) Scroll down to the table titled Vaccine Series by County of Vaccine Provider (about 2/3 down)
+    # 3) Hover over the table in the top right corner, a button should appear
+    # 4) Inspect the Network tab in the browser, then click the link
+    # 5) The token will be in the request headers under 'x-domo-embed-token`
+    token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiIxMjUyNjM5Mzc0IiwibmJ" \
+            "mIjoxNjE2NzYyOTUyLCJpc3MiOiJhcGlHYXRld2F5IiwiZW1iIjpbIntcInRva2VuXCI6" \
+            "XCIxd0I5alwiLFwibGlua1R5cGVcIjpcIlNFQVJDSEFCTEVcIixcInBlcm1pc3Npb25zXCI" \
+            "6W1wiUkVBRFwiXX0iXSwiZXhwIjoxNjE2NzkxNzYyLCJpYXQiOjE2MTY3NjI5NjIsImp0aSI6" \
+            "IjdkODBhN2YwLWZmZjEtNDc2Yi1iNGQ3LThlOTRiOWFlYzJiMiJ9.Fn5GPyRJZal_b28wA2tS" \
+            "HUBZxJv-QsbBlngmSlvmbfw"
+    def fetch(self):
+        headers = {
+            'x-domo-embed-token': self.token,
+        }
+        res = requests.post(self.csv_link, {
+            "request": json.dumps({
+                "fileName": "Vaccine Series by County of Vaccine Provider.csv",
+                "accept": "text/csv",
+                "type": "file"
+            })
+        }, headers=headers)
+        data = StringIO(res.text)
+        return pd.read_csv(data)
+
+    def normalize(self, data):
+        df = data.rename(columns={
+            "County": "location_name",
+            "Two-Dose Series Initiated": "total_vaccine_initiated",
+            "Two-Dose Series Completed": "total_vaccine_completed",
+            "Single-Dose Series Completed": "single_complete",
+            "Total Doses Administered": "total_administered"
+        })
+
+        df.total_vaccine_initiated = df.total_vaccine_initiated + df.single_complete
+        df.total_vaccine_completed = df.total_vaccine_completed + df.single_complete
+
+        crename = {
+            "total_vaccine_initiated": CMU(
```

Comment:
Again in the spirit of reducing duplication, we've been doing a few things:

1. Common CMU definitions are written in one place and then used: https://github.com/covid-projections/can-scrapers/blob/main/can_tools/scrapers/variables.py. See https://github.com/covid-projections/can-scrapers/blob/main/can_tools/scrapers/official/AL/al_vaccine.py for example
2. We've been moving this dictionary out into a class level attribute called `variables` instead of `crename` (again see https://github.com/covid-projections/can-scrapers/blob/main/can_tools/scrapers/official/AL/al_vaccine.py)
3. We have helper methods `_rename_or_add_date_and_location` and `_reshape_variables` that do most of the common transformations including parsing dates, renaming location columns, adding vintage, melting the dataframe, calling extract_CMU, etc.. Let's see if we can use those here to clean this up
4. If you use the helper methods from above, you shouldn't have to do that `cols_to_keep` stuff

Reply:
the `_rename_or_add_date_and_location` method tries to parse county names as fips numbers. Is there something missing?

Reply:
There should be two options, `location_name_column`  `location_column` - setting `location_column` will try to parse as numbers, but the `location_name_column` argument should handle county names correctly

Reply:
Fixed